serverless/functions/serverless-functions-yaml.adoc

```diff
@@ -0,0 +1,26 @@
+include::modules/serverless-document-attributes.adoc[]
+[id="serverless-functions-project-configuration"]
+= Function project configuration in func.yaml
+include::modules/common-attributes.adoc[]
+:context: serverless-functions-yaml
+
+toc::[]
+
+The `func.yaml` file contains the configuration for your function project.
+
+Generally, these values are used when you execute a `kn func` command. For example, when you run the `kn func build` command, the value in the `builder` field is used.
+
+[NOTE]
+====
+In many cases, you can override these values with command line flags or environment variables.
+====
+
+include::modules/serverless-functions-func-yaml-fields.adoc[leveloffset=+1]
+include::modules/serverless-functions-func-yaml-environment-variables.adoc[leveloffset=+1]
+
+.Additional resources
+
+* For information on overriding values in the `func.yaml` file, see xref:serverless-functions-getting-started.adoc#serverless-functions-getting-started[Getting started with functions].
```

Comment:
@rh-max Hey Max - the xrefs need to go to the top level folder and back again. https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#internal-cross-references

Could you change it in a followup PR. This is preventing the books from building on the portal.

@abrennan89 and @skrthomas please look out of this as part of reviews.

Reply:
@vikram-redhat Hey Vikram - sorry! Fixed in https://github.com/openshift/openshift-docs/pull/34630 , which also needs to be cherry-picked into `enterprise-4.6`, `enterprise-4.7`, and `enterprise-4.8`.